README.md

```diff
@@ -34,14 +34,28 @@ const bytesWritten = await zipper.finalize()
 // bytesWritten -> 276
 ```
 
-## Testing
+## Options
+
+Options can be passed as the second argument to `addDir`:
+
+```js
+await zipper.addDir('directory', {
+  ignoreHidden: true,
+  exclude: ['/tmp/']
+})
+```
 
-Run `npm test` to run a simple test suite; this tests that no errors occur, but
-it does not test that the file is created correctly. Please manually inspect
-`test/output.zip` and ensure it matches the files in `test/fixtures`.
+`ignoreHidden`: whether to ignore hidden files (default: `false`)
+
+`exclude`: array of paths to exclude (default: `[]`)
+
+_Note_: this applies to files as well as directories.
+
+## Testing
 
-Admittedly it's a down and dirty test, but it is enough to exercise the library
-properly and it also provides a sample usage.
+Run `npm test` to run a simple test suite; this tests that no errors occur and
```

Comment:
Not quite simple as it was, could possibly use a better adjective.

Reply:
```suggestion
Run `npm test` to run the test suite. The tests insure that no errors occur and
```